draft-ietf-rtcweb-jsep.xml

```diff
@@ -1378,10 +1384,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             setLocalDescription. However, the remote side, either
             before or after setRemoteDescription, decides it does not
             want to accept the new parameters, and sends a reject
-            message back to the offerer. Now, the offerer, and possibly
-            the answerer as well, need to return to a stable state and
-            the previous local/remote description. To support this, we
-            introduce the concept of "rollback".</t>
+            message back to the offerer.
```

Comment:
no - change - it was changed , but EKR thought orig was better so I changed it back. Next rer will try and line up formatting to make it clear there is not change.